### PR TITLE
load_to_bigquery: Enable to set properties at inifile or pbc

### PR DESCRIPTION
**Pull request by tanak**
you can set the following propeties at inifile or pbc.
project_id
dataset

### CODE REVIEW

File: plugins/patriot-gcp/lib/patriot_gcp/command/load_to_bigquery.rb [hunk 1]
```diff
@@ -4,13 +4,27 @@ class LoadToBigQueryCommand < Patriot::Command::Base
...
+        @service_account  = ini["gcp"]["service_account"]
+        @private_key      = ini["gcp"]["private_key"]
+        @key_pass         = ini["gcp"]["key_pass"]
+        @project_id     ||= ini["bigquery"]["project_id"]
+        @dataset        ||= ini["bigquery"]["dataset"]
+      end
+
       def job_id
-        job_id = "#{command_name}_#{@dataset}_#{@table}"
+        set_conf
```
Comment (zenmyo) [inline plugins/patriot-gcp/lib/patriot_gcp/command/load_to_bigquery.rb]: set_conf should be invoked in configure.
(e.g., https://github.com/CyberAgent/patriot-workflow-scheduler/blob/develop/lib/patriot/command/sh_command.rb#L36)

**Comment by tanak** [comment]
Thank you for your advice.
Could you check the modifications again?

**Comment by zenmyo** [comment]
+1, LGTM

**Comment by tsuda7** [comment]
Thx!

**Pull request merged**